app/src/main/java/com/example/ncov19traking/ui/CovidCustomCard.kt

```diff
@@ -0,0 +1,98 @@
+package com.example.ncov19traking.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import com.example.ncov19traking.R
+import com.google.android.material.card.MaterialCardView
+
+private const val DEFAULT_NUMBER_TEXT = "0"
+private const val DEFAULT_PERCENTAGE_TEXT = "0%"
+
+class CovidCustomCard @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : MaterialCardView(context, attrs, defStyleAttr) {
+
+    private val defaultTitleText = resources.getString(R.string.custom_covid_card_default_title)
+    private var titleText = defaultTitleText
+    private var numberText = DEFAULT_NUMBER_TEXT
+    private var percentageText = DEFAULT_PERCENTAGE_TEXT
+    private var numberTextColor = resources.getColor(R.color.recoveredColor)
+    private var percentageTextColor = resources.getColor(R.color.textColor)
+    private var titleTextView: TextView? = null
+    private var numbersTextView: TextView? = null
+    private var percentageTextView: TextView? = null
+
+    init {
+        receivingAttributes(context, attrs, defStyleAttr)
```

Comment:
often method naming is done in infinitive form like `receiveAttributes`, `receive`, `parse`, `get`, not `receiving`, `parsing` or `getting`